ui/static/ui/js/learning_resources.jsx

```diff
@@ -62,7 +62,10 @@ define('learning_resources', [
 
       var slug = this.props.selectedVocabulary.slug;
 
-      return <div className="col-sm-6">
+      return <div className="form-group">
+        <label htmlFor="vocabChooser" className="col-sm-4 control-label">
```

Comment:
`for` matches a label with an id defined in the `id` attribute. `key` is only used in React to make it easier to diff two React components, and usually only within a collection of similar elements. You will also need to make sure `id` won't conflict with other elements, probably by concatenating the vocab slug or id number.